### PR TITLE
Added allowlisted domains

### PR DESCRIPTION
**Pull request by ianf77**
Added list of allowlisted domains and other requirements

Add in the doc from config AAP the domains which need to be whitelisted

https://issues.redhat.com/browse/AAP-4122

### CODE REVIEW

**Comment by ianf77** [comment]
[@alfferna]
Does this look correct

**Review by ariordan-redhat — COMMENTED** [review]
@ianf77 
A couple of suggestions.
I don't know whether the SME has access to our repo: it might be useful to post a  preview on Fileshare and link to the modified sections.

File: downstream/modules/automation-hub/proc-create-synclist.adoc [hunk 1]
```diff
@@ -12,11 +12,21 @@ By default, all Red Hat Certified collections are included in your initial organ
 
 * You have a valid Ansible Automation Platform subscription.
 * You have Organization Administrator permissions for cloud.redhat.com.
+* Ensure that the following domain names are part of either the firewall or the proxy's allowlist for successful connection and download of collections from Automation hub or Galaxy server:
+** `galaxy.ansible.com`
+** `cloud.redhat.com`
+** `console.redhat.com`
+** `sso.redhat.com`
+* The contents of the Automation Hub are stored in `S3`, ensure the following domain name is also part of the allow list:
+** `automation-hub-prd.s3.us-east-2.amazonaws.com`
```
[the remark this answers is not in the document]
Comment (ianf77) [reply downstream/modules/automation-hub/proc-create-synclist.adoc:21]: @ariordan-redhat , we could change it to <AWS region> ?

[the remark this answers is not in the document]
Comment (ariordan-redhat) [reply downstream/modules/automation-hub/proc-create-synclist.adoc:21]: @ianf77 I think we'd need SME input on whether this AWS region will always be used.

[the remark this answers is not in the document]
Comment (ianf77) [reply downstream/modules/automation-hub/proc-create-synclist.adoc:21]: us-east-2 is the account set up specifically for this. Unlikely to change according to Alfonso.

**Pull request merged**